tutorials/Tutorial8_Preprocessing.ipynb

```diff
@@ -0,0 +1,424 @@
+{
```

Comment:
<p>How about adding one last section here "Bringing everything together", where you show in one cell the whole flow from fileconversion -&gt; preprocessing -&gt; calling document_store.write_documents()? As a user, I would really like to have such a code snippet that I can easily copy over to my own scripts...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack/pull/706/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>